Resources/Private/Frontend/JavaScript/Features/ContentTree.ts

```diff
@@ -20,10 +20,10 @@ function buildContentTree() {
   if (iframe == null) {
```

Comment:
I would suggest using the strict type check with `===` or even check against a falsey value.

So either `if(iframe === null)` or `if(!iframe)`. However, the second option also checks against values like `undefined`, which can't really be delivered by the `querySelector` function